sa/_db/migrations/20210924100000_OldFQDNSets.sql

```diff
@@ -1,24 +0,0 @@
--- TODO(#5670): Remove this file and the _db-next pointer to it.
-
--- +goose Up
--- SQL in section 'Up' is executed when this migration is applied
-
-CREATE TABLE `fqdnSets_old` (
-  `id` bigint(20) NOT NULL AUTO_INCREMENT,
-  `setHash` binary(32) NOT NULL,
-  `serial` varchar(255) NOT NULL,
-  `issued` datetime NOT NULL,
-  `expires` datetime NOT NULL,
-  PRIMARY KEY (`id`),
-  UNIQUE KEY `serial` (`serial`),
-  KEY `setHash_issued_idx` (`setHash`,`issued`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8;
-
-ALTER TABLE fqdnSets DROP INDEX IF EXISTS serial, ADD INDEX serial (serial);
-ALTER TABLE fqdnSets PARTITION BY RANGE(id) (
-    PARTITION p_start VALUES LESS THAN MAXVALUE);
-
```

Comment:
Doesn't this part need to get migrated to the merged schema?

Reply:
This is already in the partitioning migration: https://github.com/letsencrypt/boulder/blob/23fc3c907b71e6903fbd8ab5dc77b2269a755783/sa/_db/migrations/20210308140000_SimplePartitioning.sql#L14-L16